### DISCUSSION TITLE
DAI 2023-2024 - My Docker Compose application - Jeremiah Steiner

### DISCUSSION THREAD

**Discussion by BetterBlood** [heig-vd-dai-course/heig-vd-dai-course#124] [Show and tell]
[The link to your GitHub repository with a screenshot of your Docker image published on GitHub Container Registry](https://github.com/BetterBlood/my-custom-dockerfile)
![image](https://github.com/heig-vd-dai-course/heig-vd-dai-course/assets/45463619/c4f918d5-c5e2-4010-ab13-660c6b2733f4)

**Comment by ludelafo** [comment]
Everything seems good to me, congrats!
